types/shelljs/index.d.ts

```diff
@@ -50,7 +51,7 @@ export function find(...path: Array<string | string[]>): ShellArray;
  * @param source  The source.
  * @param   dest  The destination.
  */
-export function cp(source: string | string[], dest: string): void;
+export function cp(source: string | string[], dest: string): ShellArray;
```

Comment:
I think it is a `ShellString`, not a `ShellArray`.
Could you also add a `@return` annotation?